### DISCUSSION TITLE
Wizard Step OnClick

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#4113] [Help]
Is there any way to associate the Wizard's NEXT button with an onclick or wire:click event?

**Comment by danharrin** [comment]
What are you trying to do?

**Reply by jcgmalave** [reply]
The problem is that within a wizard-type form in a step I have within a ViewField some inputs with a default value, when pressing next to go to the next step or pressing save those values ​​are reset and go blank

**Reply by danharrin** [reply]
That sounds like a problem with your ViewField rather than the wizard, and I don't know how attaching a wire:click would help that

**Comment by MarJose123** [comment]
In related to this, is it possible to attach `wire:click` or similar, I have requirement that I need to check if the data inputted is valid by calling external API and if not valid I need the wizard to step back.
